### ISSUE TITLE
Feature: 123 Students fail to find mentoring schedule

### ISSUE THREAD

**Issue by madisavage** [rit-sse/WebsiteTheSSEquel#585]
<img width="832" height="574" alt="Image" src="https://github.com/user-attachments/assets/604c6c7b-6643-4cd3-961d-96fe7a0e5f1c" />
Maybe make the mentoring hours button on the front page link obviously to the schedule?
